meta-mender-core/recipes-bsp/grub-mender-grubenv/grub-mender-grubenv_1.1.0.bb

```diff
@@ -60,13 +79,37 @@ mender_grub_storage_device=$mender_grub_storage_device
 kernel_imagetype=${KERNEL_IMAGETYPE}
 EOF
 
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'mender-uuid', 'true', 'false', d)}; then
+        if ${@bb.utils.contains('DISTRO_FEATURES', 'mender-image-gpt', 'true', 'false', d)} || ${@bb.utils.contains('DISTRO_FEATURES', 'mender-image-uefi', 'true', 'false', d)}; then
+            mender_rootfsa_uuid=${MENDER_ROOTFS_PART_A_UUID}
+            mender_rootfsb_uuid=${MENDER_ROOTFS_PART_B_UUID}
+        else
+            mender_rootfsa_uuid="${MENDER_DISK_IDENTIFIER}-${@get_part_nummber_padded_from_device(MENDER_ROOTFS_PART_A)}"
+            mender_rootfsb_uuid="${MENDER_DISK_IDENTIFIER}-${@get_part_nummber_padded_from_device(MENDER_ROOTFS_PART_B)}"
+        fi
+
+        cat >> ${B}/mender_grubenv_defines <<EOF
+mender_rootfsa_uuid=$mender_rootfsa_uuid
+mender_rootfsb_uuid=$mender_rootfsb_uuid
+EOF
```

Comment:
Following my earlier comment about setting

```
MENDER_ROOTFS_PART_A="/dev/disk/by-uuid/12345678-1234-5678-9abc-56789abcdef"
```

I believe the need for `mender_rootfsa_uuid` and `mender_rootfsb_uuid` falls away, and we can use

```
mender_rootfsa_part="PARTUUID=12345678-1234-5678-9abc-56789abcdef"
```

directly instead.